File: GettingStarted/gradingSystem.py
n= int(input())

def gradingSystem(n):
    if n > 90:
        print("excellent")
        
    elif n > 80 and n <= 90:
        print("good")
        
    elif n > 70 and n <= 80:
        print("fair")
        
    elif n > 60 and n <= 70:
        print("meets expectations")
        
    else:
        print("below par")
gradingSystem(n)

File: GettingStarted/gradingSystem.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      // input - don't change this code
      Scanner scn = new Scanner(System.in);
      int marks = scn.nextInt();
      // input - don't change this code
      gradingSystem(marks);
    }
      // code here 
      public static void gradingSystem(int n){
        if(n > 90){
            System.out.print("excellent");
        }
        else if(n > 80 && n <= 90){
            System.out.print("good");
        }
        else if(n > 70 && n <= 80){
            System.out.print("fair");
        }
        else if(n > 60 && n <= 70){
            System.out.print("meets expectations");
        }
        else{
            System.out.print("below par");
        }
     }
  
   }